docs/agent-machine-setup-windows.md

```diff
@@ -0,0 +1,138 @@
+# Setting up a new Crater Windows agent machine
+
+## Operating System
+
+The agent machine should run Windows Server build 1803 (Windows 10 build 1803
```

Comment:
I think it's a good idea to mention up here that Windows 10 Build 1809 and above will work.

Reply:
There's now an officially published [1803 image] that's hard-coded as the default, so I think the mention down below is sufficient. I meant for this paragraph to be the bare minimum of information for someone to get up and running with the stock set up, and if you wanna use 1809 you'll need to build your own docker image and pass it to `crater`.

Reply:
Maybe I need to add, "later versions of Windows will work, but you'll need to use a custom Docker image, see below".

Reply:
Yea that's enough.